lightspeed/modules/proc_create-connection-secrets.adoc

```diff
@@ -0,0 +1,89 @@
+:_content-type: PROCEDURE
+
+[id="create-connection-secrets_{context}"]
+
+= Creating connection secrets
+
+Use this procedure to create secrets to connect to {PlatformName} and {ibmwatsonxcodeassistant}.
+
+.Prerequisites
+* You have installed the {PlatformNameShort} operator on the {OCP}. 
+* You have created an OAuth application in the automation controller.
+* You have obtained an API key and a model ID from {ibmwatsonxcodeassistant}. 
++
+For information about obtaining an API key and model ID from {ibmwatsonxcodeassistant}, see the link:https://cloud.ibm.com/docs/watsonx-code-assistant[{ibmwatsonxcodeassistant} documentation].
+
+.Procedure
+. Go to the {OCP}. 
+. Select *Workloads → Secrets*.
```

Comment:
For consistency throughout our docs, use the asciidoc menu macros for navigation instructions: https://docs.asciidoctor.org/asciidoc/latest/macros/ui-macros/
So it should look like Select menu:Workloads[Secrets]

Reply:
Done